docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -170,7 +178,7 @@ Unit tests provide automated software testing during your development and publis
 
    ![Console window showing the word olives searched in the string, 'Iro ate olives by the lake, and the olives were wonderful.' The app responds, 'The search word olives appears 2 times.'](./media/using-on-mac-vs-full-solution/vsmacfull14.png)
 
-1. The last feature to explore is debugging with Visual Studio for Mac. Set a breakpoint on the `Console.WriteLine` statement: Select in the left margin of line 23, and you see a red circle appear next to the line of code. Alternatively, select anywhere on the line of code and select **Run > Toggle Breakpoint** from the menu.
+1. The last feature to explore is debugging with Visual Studio for Mac. Set a breakpoint on the `Console.WriteLine` statement: Select in the left margin of line 23, and you see a red circle appear next to the line of code. Alternatively, select anywhere on the line of code and select **Run** > **Toggle Breakpoint** from the menu.
 
    ![Breakpoint is set on line 23, the Console.WriteLine statement](./media/using-on-mac-vs-full-solution/vsmacfull15.png)
 
```

Comment:
It's not relevant from a branding perspective since "Xamarin Studio" doesn't match, and I don't trust it. It was produced >3 months ago at the same time we did this topic. It's a safe bet that parts of it are dated. I'll try this round with a "See also" to the release notes, which do provide good detail on the feature set of this release.